ideascube/serveradmin/models.py

```diff
@@ -0,0 +1,27 @@
+from django.conf import settings
+from django.db import models
+
+from ideascube.models import JSONField
+
+
+class Setting(models.Model):
+    class Meta:
+        ordering = ['-date']
+
+    namespace = models.CharField(max_length=40)
+    key = models.CharField(max_length=40)
+    value = JSONField()
+    actor = models.ForeignKey(settings.AUTH_USER_MODEL)
+    date = models.DateTimeField(auto_now=True)
+
+    def as_boolean(self):
```

Comment:
I really wouldn't like using `**kwargs` here.

`**kwargs` generally implies something like « pass in what you want, I'll make sense of it »

Here, we expect a single optional argument, `default`.

If `None` as a default value ever becomes a concern, then we could just introduce a new « empty » value, and check for that instead of None.

But I feel that's a concern better kept for later, if that ever happens.